.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
 python:
   - "2.7"
+  - "3.4"
```

Comment:
I don't think we should support 3.2 because very few people use it and 3.2 compatibility is more complicated: the `u'unicode string'` was removed in 3.0 and only [readded](https://www.python.org/dev/peps/pep-0414/) in 3.3.

Given that travis relies on volunteers for computing resources, I don't think we should have a build for every version. I think 2.7, 3.3, and nightly should be enough, and maybe 2.6 if that is necessary.